src/core/integrate.cpp

```diff
@@ -576,12 +543,12 @@ void rescale_forces_propagate_vel() {
 #ifdef NPT
         if (integ_switch == INTEG_METHOD_NPT_ISO &&
             (nptiso.geometry & nptiso.nptgeom_dir[j])) {
-          nptiso.p_vel[j] += Utils::sqr(p.m.v[j]) * p.p.mass;
-            p.m.v[j] += p.f.f[j] + friction_therm0_nptiso(p.m.v[j]) / p.p.mass;
+            nptiso.p_vel[j] += Utils::sqr(p.m.v[j] * time_step) * p.p.mass;
```

Comment:
This looks strange: `[p_vel] = L^2 * M`? If this compensated by some coefficient?

Reply:
```p_vel``` gets divided by ```time_step * time_step``` in https://github.com/KaiSzuttor/espresso/blob/3342bd04b01a783bd053b74a922e87f039001b0a/src/core/integrate.cpp#L576

Reply:
Ok